pkgs/top-level/python-packages.nix

```diff
@@ -15023,11 +15023,13 @@ let
 
   wxPython = self.wxPython28;
 
-  wxPython28 = callPackage ../development/python-modules/wxPython/2.8.nix {
+  wxPython28 = import ../development/python-modules/wxPython/2.8.nix {
```

Comment:
Why did you get rid of `callPackage`? that removes ability to override derivation or the function